host-bin/edit-chroot

```diff
@@ -403,7 +403,7 @@ for NAME in "$@"; do
             tardest="$dest"
             splitpid=''
         fi
-        echo -n "  Backing up $CHROOT to $dest..." 1>&2
+        echo -n "  Backing up $CHROOT to $(readlink -m $dest)..." 1>&2
```

Comment:
Readlink should look like `$(readlink -m -- "$dest")`
...that is, with double-quotes and the -- parameter.